src/main/java/george/backend/hiring/JavaApplication.java

```diff
@@ -0,0 +1,9 @@
+package george.backend.hiring;
+
+public class JavaApplication {
+
+    public static void main(String[] args) {
+        System.out.println("Hello, World!");
```

Comment:
ℹ️ Main method will be needed anyway. Is usually added by project stub generators.

Reply:
**[CR]** The class intentionally had no `main`
Reasons:
- we had candidates who did not know how to write a `main` even with google
- for some candidates having two `main` methods confused them and their IDE setups

Reply:
I am not sure that we need to be optimizing for those cases, and that having the `main` there would save that particular candidate. 😄 

But sure, I can remove it.